landing-pages/site/content/en/blog/announcing-new-website.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "New Airflow website"
+linkTitle: "New Airflow website"
+author: "Aizhamal Nurmamat kyzy"
+description: "We are thrilled about our new website!"
+tags: ["Community"]
+date: "2019-12-11"
+---
+
+The brand [new Airflow website](https://airflow.apache.org/) has arrived! Those who have been following the process know that the journey to update [the old Airflow website](https://airflow.readthedocs.io/en/1.10.6/) started at the beginning of the year. 
+Thanks to sponsorship from the Cloud Composer team at Google that allowed to 
```

Comment:
```suggestion
Thanks to sponsorship from the Cloud Composer team at Google that allowed us to 
```
I am not an expert but I looked at the internet.
https://ell.stackexchange.com/a/229555
@mschickensoup WDYT?